lib/crypto/packaged.py

```diff
@@ -103,8 +103,8 @@ def _sign_app(src, dest, ids, reviewer, tempname, local=False):
         storage = local_storage
 
     with local_storage.open(tempname) as temp_f, \
-            storage.open(dest, 'w') as dest_f:
-        shutil.copyfileobj(temp_f, dest_f)
+            storage.open(dest, 'w') as dst_f:
+        shutil.copyfileobj(temp_f, dst_f)
```

Comment:
Good call, I wanted to do a pass on `copyfileobj` usage but forgot, I'll take a look.